src/azul/service/catalog_controller.py

```diff
@@ -41,14 +42,13 @@ def list_catalogs(self) -> schema.object(
                 catalog.name: {
                     'internal': catalog.is_internal,
                     'atlas': catalog.atlas,
-                    'plugins': [
-                        {
-                            'type': plugin_type,
-                            'name': plugin,
+                    'plugins': {
+                        plugin_type: {
```

Comment:
The only way that could be true is if `"name"` were the value of `plugin_type` which it can't be. The only possible values for `plugin_typ` are  `"metadata"` or `"repository"`.